scholarly/_scholarly.py

```diff
@@ -255,13 +258,29 @@ def citedby(self, object: Publication)->_SearchScholarIterator:
         :param object: The Publication object for the bibtex exportation
         :type object: Publication
         """
-        if object['container_type'] == "Publication":
-            publication_parser = PublicationParser(self.__nav)
-            return publication_parser.citedby(object)
-        else:
+
+        if object['container_type'] != "Publication":
             self.logger.warning("Object not supported for bibtex exportation")
             return
 
+        if object["bib"]["citedby"] < 999:
+            return PublicationParser(self.__nav).citedby(object)
+        else:
+            try:
+                year_low = int(object["bib"]["pub_year"])
+                year_end = int(datetime.date.today().year)
+            except:
```

Comment:
Don't have blank `except` clauses. What execptions did you particularly anticipate here?

Reply:
Gotcha, I was trying to catch the rare case where `pub_year` is not found.